cmd/podman/create.go

```diff
@@ -555,16 +555,15 @@ func parseCreateOpts(ctx context.Context, c *cli.Context, runtime *libpod.Runtim
 		workDir = data.ContainerConfig.WorkingDir
 	}
 
-	// ENTRYPOINT
-	// User input entrypoint takes priority over image entrypoint
-	entrypoint := c.StringSlice("entrypoint")
-	if len(entrypoint) == 0 && data != nil {
+	var entrypoint []string
+	if c.IsSet("entrypoint") {
+		// ENTRYPOINT
+		if err := json.Unmarshal([]byte(c.String("entrypoint")), &entrypoint); err != nil {
+			return nil, errors.Wrapf(err, "unable to translate entrypint %s", c.String("entrypoint"))
```

Comment:
"entrypint" to "entrypoint"

Reply:
Does this work for single values? I tried unmarshalling a string into a []string in playground and it didn't work. We can't require all entrypoints to be arrays.